.pre-commit-config.yaml

```diff
@@ -7,8 +7,8 @@ repos:
     hooks:
       - id: licenseheaders
         args: ["--tmpl=LICENSE_SHORT", "--ext=py", "-f"]
-  - repo: https://github.com/ambv/black
-    rev: 22.1.0
+  - repo: https://github.com/psf/black
```

Comment:
I have no context and am just curious: why did we switch maintainers?  A quick look at both GitHub pages didn't give me a clear answer

Reply:
`https://github.com/ambv/black` is now redirecting to `https://github.com/psf/black` so it's not an actual maintainer switch, arguably unnecessary but just for clarity.

Reply:
Cool.  Cool cool cool. thanks for the context